Src/Newtonsoft.Json/JsonTextReader.cs

```diff
@@ -445,7 +445,7 @@ public override string ReadAsString()
         /// <summary>
         /// Reads the next JSON token from the stream as a <see cref="Byte"/>[].
         /// </summary>
-        /// <returns>A <see cref="Byte"/>[] or a null reference if the next JSON token is null. This method will return <c>null</c> at the end of an array.</returns>
+        /// <returns>A <see cref="Byte"/>[] or <c>null</c> if the next JSON token is null. This method will return <c>null</c> at the end of an array.</returns>
```

Comment:
Here's an instance of "null" that I chose not to markup with `<c>`. I'm not entirely sure about Json.NET's own concept of nullity, but after running a quick test, I suspect that this phrase "if the next JSON token is null" actually means "if the next JSON token has a type of either `JsonToken.Null` or `JsonToken.None`".